vim_menutrans_helper.py

```diff
@@ -0,0 +1,196 @@
+import re
+import glob
+
+menu_item = {"me", "menu", "noreme", "noremenu", "am",
+             "amenu", "an", "anoremenu", "nme", "nmenu",
+             "nnoreme", "nnoremenu", "ome", "omenu", "onoreme",
+             "onoremenu", "vme", "vmenu", "vnoreme", "vnoremenu",
+             "xme", "xmenu", "xnoreme", "xnoremenu", "sme", "smenu",
+             "snoreme", "snoremenu", "ime", "imenu", "inoreme", "inoremenu",
+             "cme", "cmenu", "cnoreme", "cnoremenu", "tlm", "tlmenu", "tln",
+             "tlnoremenu", "E330", "E327", "E331", "E336", "E333", "E328", "E329",
+             "E337", "E792", "tunmenu", "tu", "popup", "popu"}
+toolbar_item = {"tmenu", "tm"}
+
+# unescape double quotes
+def unescape_double_quotes(origin):
+    origin = origin.replace('\\"', '"')
+    origin = origin.replace("\\'", "'")
+    origin = origin.replace('\\t', '\t')
+    origin = origin.replace('\\\\', '\\')
+    return origin
+
+# unescape single quotes
+def unescape_single_quotes(origin):
+    origin = origin.replace("''", "'")
+    return origin
+
+# get the diffence between untranslated file and translated file
+def makeUnTranslatedDic(untranslated_file, untranslated_dict):
+    # deal with the file need to be translated
+    with open(untranslated_file, encoding="latin1") as f1:
+        for line_number, line in enumerate(f1):
+            tobe_translated_words = []
+
+            line = line.strip()
+            line = line.rstrip()
```

Comment:
`strip()` removes both leading and trailing white spaces. There's no need to `rstrip()` after that - you're not supposed to have any trailing spaces at all after `strip()`.